src/main/java/com/flab/bbt/auth/service/AuthService.java

```diff
@@ -18,13 +18,18 @@ public AuthService(UserRepository userRepository, PasswordEncrypter passwordEncr
     }
 
     public void signUp(User user) {
-        // [ToDo]이메일 중복체크
-
+        if(isEmailExist(user.getEmail())){
```

Comment:
@hoonpark7442 
(minor) (제가 영어를 잘 하는편은아니지만..)
좋은네이밍은 개발자를 항상 따라따라다니는 숙제입니다.
이처럼 중복체크하는 로직도 자주 사용되죠.
exists는 존재여부를 체크할때 사용하는데 자동사라 is와 같이 사용하지는 않습니다. 
중복검사라는 의미를 담아 `is duplicated email` 정도로 쓰는건 어떨까요?
@jungin-jin-choi 님은 어떻게 생각하시나요?

Reply:
```suggestion
        if(hasDuplicateEmail(user)){
```
어떤가욤?
하하 저도 네이밍 항상 엄청 어려워해서 각종 구글링/깃헙서치 엄청 했었는데....
답이 없는 이 주제에 대해서 제가 내린 저만의 결론??은
`boolean 메서드명(파라미터)`라는 메서드가
`if(메서드명(파라미터))` 로 활용될 것이라면
_`"if 파라미터 메서드명"`_ 가 natural language 처럼 읽히면 좋다! 입니당.
_`if user has duplicate email`_ 처럼 읽히면 자연스러우니깐ㅎㅎ 

참고로 변수명이랑 메서드명이랑 또 다른 것 같습니다.
if(isUserExists == true) / if hasDuplicate(user)
요런 느낌이랄까요 ??

Reply:
@f-lab-lion @jungin-jin-choi 
네이밍 수정했습니다. 중복이메일 체크용 메서드에 user 객체 전부 파라미터로 담아서 넘겨줘도 되나 싶어서 일단은 `isDuplicatedEmail(String email)`로 메서드명을 수정했습니다. 유저 객체를 전부 담아도 큰 상관은 없는걸까요? 궁금하네여